tests/test_pay.py

```diff
@@ -2645,3 +2645,76 @@ def test_partial_payment(node_factory, bitcoind, executor):
     assert pay['status'] == 'complete'
     assert pay['number_of_parts'] == 2
     assert pay['amount_sent_msat'] == Millisatoshi(1002)
+
+
+@unittest.skipIf(not EXPERIMENTAL_FEATURES, "needs parallel_id support")
+def test_partial_payment_timeout(node_factory, bitcoind):
+    l1, l2 = node_factory.line_graph(2)
+
+    inv = l2.rpc.invoice(1000, 'inv', 'inv')
+    paysecret = l2.rpc.decodepay(inv['bolt11'])['payment_secret']
+
+    route = l1.rpc.getroute(l2.info['id'], 500, 1)['route']
+    l1.rpc.call('sendpay', [route, inv['payment_hash'], None, 1000, inv['bolt11'], paysecret, 1])
```

Comment:
The `None` to skip an unassigned argument is hard to read. Could use the `kwargs` dict instead which is also nice self-documenting.

Reply:
Well, it'll go away once sendpay arg is no longer experimental anyway, so hard to really mind.